x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings_by_resource/resource_findings/use_resource_findings.ts

```diff
@@ -59,6 +59,9 @@ const getResourceFindingsQuery = ({
     sort: [{ [sort.field]: sort.direction }],
     aggs: {
       ...getFindingsCountAggQuery(),
+      cloudAccountName: {
```

Comment:
nit: I prefer here we would use the `belong_to` runtime field.
Just in case findings might have the same account name but for some weird reason, different clusterId (which suppose to be empty). So in that kind of edge case, you will have another entry for the same resource :\

Reply:
The problem with that is that we will also have to change the title to `Belongs To` in the shared values area, and this name usually comes with a tooltip to explain itself. I think in this case its better to have both fields so we can either display `Cluster ID` or `Cloud Account Name` depending on which on of those fields has data in it.
There are more options to understand which finding type we are working with but i think what was implemented here is the most straight forward

Reply:
The problem with that is that we will also have to change the title to `Belongs To` in the shared values area, and this name usually comes with a tooltip to explain itself. I think in this case its better to have both fields so we can either display `Cluster ID` or `Cloud Account Name` depending on which on of those fields has data in it.
There are more options to understand which finding type we are working with but i think what was implemented here is the most straight forward

Reply:
I understand, but here you are breaking the meaning of the query in order to understand the `posture_type`. We should aggregate by `posture_type` and use that as our way to differentiate the data types, the same thing you would do with `findings`. Don't you think?

Reply:
I think this might add quite a bit of complicity to this task, depending on what we want here for the long term. but personally, unless we need to display both kspm and cspm resources on the same screen I wouldn't go for the effort of adding aggregations between the type in this stage.